ddtrace/span.py

```diff
@@ -262,7 +264,7 @@ def set_tag(self, key, value=None):
                 pass
 
         # Set integers that are less than equal to 2^53 as metrics
-        if val_is_an_int and abs(value) <= 2 ** 53:  # type: ignore[arg-type]
+        if value and val_is_an_int and abs(value) <= 2 ** 53:
```

Comment:
Use `value is not None` to be explicit, unless you want to ignore `0` which would be weird IMHO